jscomp/ml/translcore.ml

```diff
@@ -350,6 +370,19 @@ let primitives_table =
       ("%lefloat", Pfloatcomp Cle);
       ("%gtfloat", Pfloatcomp Cgt);
       ("%gefloat", Pfloatcomp Cge);
+      ("%negbigint", Pnegbigint);
+      ("%addbigint", Paddbigint);
+      ("%subbigint", Psubbigint);
+      ("%mulbigint", Pmulbigint);
+      ("%divbigint", Pdivbigint Safe);
```

Comment:
Looks like there's no way to generate an unsafe bigint div, as `Pdivbigint` always receives `Safe`?
If that's true and intended, then I guess `Pdivbigint` can be simplified and the runtime does not need to take a parameter.

Reply:
Removed https://github.com/rescript-lang/rescript-compiler/pull/6670/commits/649dc3e170b8316bcf5ef794dc6de69cec9699f2